canonical/vanilla-framework
# Add sliding nav on mobile

## Done

Added a sliding animation for nested nav elements

Fixes https://warthogs.atlassian.net/browse/WD-2617

## QA

https://vanilla-framework-4767.demos.haus/docs/examples/patterns/navigation/sliding-search
https://vanilla-framework-4767.demos.haus/docs/examples/patterns/navigation/sliding-light
https://vanilla-framework-4767.demos.haus/docs/examples/patterns/navigation/sliding-dark

- check the demo on large and small screens
- try keyboard navigation too

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

![slidiiningin](https://github.com/canonical/vanilla-framework/assets/11927929/2dad8ff0-82f8-421a-a90a-b111d3a61e3b)
